cloud: add compression formats (#13447)
pingcap/docs: tidb-cloud/import-sample-data.md

Reviewer: <div label="GCP"> also needs similar updates for the **Data format** field description

    Fill in the following parameters:

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.

Author: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.

Author: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can name the file as `bikeshare.trips.sql.gz`.

Reviewer 2: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.

Reviewer 2: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.

Author: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.

Author: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.

Author: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database and table schema files.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.

Reviewer 2: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```

>     - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.

Reviewer: ```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which `${suffix}` is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```
